src/test-run/commands/options.js

```diff
@@ -254,3 +256,20 @@ export class AssertionOptions extends Assignable {
         ];
     }
 }
+
+// Press
+export class PressOptions extends ActionOptions {
+    constructor (obj, validate) {
+        super();
+
+        this.confidential = void 0;
```

Comment:
`void 0` is necessary as a default value. It's needed to indicate that a user didn't set this flag explicitly. 

I've tried to use `enum` for this flag, but it looks overcomplicated in my opinion (ts-hack to create 3-state boolean):
```ts
enum ConfidentialSetting {
    False,
    True,
    CheckRequired
}
```
Now by default, this option will be shown in the "options" property of the command: 
```js
{
    type:    TYPE.typeText,
    options: {
        confidential: 2 // or the "CheckRequired" string (not better)
    }
}
```
Whereas `void 0` wouldn't be set at all. I think it's better to stick with the `void 0` variant.

Reply:
It's not clear what the `void 0` value means in the context of the `confidential` option.
So, in my opinion, the `enum ConfidentialSetting` is a good variant.

@AndreyBelym what do you think about it?

Reply:
IMHO:
TLDR; I'm against a tri-state enum:

1. In this enum, 2 of 3 values just replicate boolean options. For me, it's a sign of redundant complexity. 

2. I can't see any case when I would want to specify `ConfidentialSetting.CheckRequired` manually. Why should we give it a name, if a user won't interact with it?

3. I can describe easily the boolean configuration:
   If `true`, we hide the input value
   If `false`, we show the input value
   By default, it's `true` when the target selector is input[type=password], `false` otherwise.

Reply:
Well, about point 2: I see a case, if we implement some global-level options, you may want to reset this flag to default in some cases. But since `undefined` is a common way to reset something to default in JS everywhere, I don't see it as a problem.

Reply:
Ok. Let it be.